public/js/creatures.js

```diff
@@ -0,0 +1,61 @@
+const { Router } = require('express');
```

Comment:
This whole file should live inside your lib folder..
Something like 
lib/routes/creatures.js

Reply:
you're an angel, thank you! fixed!